contrib/babelfishpg_tsql/src/iterative_exec.c

```diff
@@ -1230,6 +1228,14 @@ int dispatch_stmt_handle_error(PLtsql_execstate *estate,
 
 		MemoryContextSwitchTo(cur_ctxt);
 
+		/* In case of RO functions we want the caller to handle errors. */
+		if (ro_func)
```

Comment:
we want to validate that error handing inside functions is same as that of SQL server. Given that no test has failed, we are missing that coverage. We should cover cases where a statement inside function throws various error codes like
- stmt terminating error
- batch terminating error
- transaction rollback error

We should also cover xact_abort ON/OFF, try/catch etc. The behaviour should match with SQL server.

In addition, there should be some generic testing where
func1 -> func2 -> func3 -> error
proc1 -> proc2 -> func1 -> error
trigger -> func1 -> error
tsqlfunc -> pgfunc -> error
pgfunc -> tsqlfunc -> error

Some of these might already be there but we should validate the same.